src/components/expandable/expandable.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactNode, useState } from 'react';
+import styled, { css } from 'styled-components';
+import ChevronSVG from './expandable-chevron.svg';
```

Comment:
Time flies, we have now an icon component. Please use the right icon instead of importing the file. You can do this after this PR.
![Screen Shot 2021-12-06 at 15 24 40](https://user-images.githubusercontent.com/1701755/144863159-c16e434d-aea1-4241-833c-45c7ef97fc99.png)